Can't pass relative path to module on command line

If I'm in `$DIR` and my code is in `$DIR/src`, this doesn't work:

```bash
cd $DIR
pulumi preview ./src
```

The error is:
``` 
info: Error: Cannot find module 'src'
info:     at Function.Module._resolveFilename (module.js:469:15)
info:     at Function.Module._load (module.js:417:25)
info:     at Module.require (module.js:497:17)
info:     at require (internal/module.js:20:19)
```

This works:
```bash
pulumi preview $DIR/src
```

--- a/cmd/destroy.go
+++ b/cmd/destroy.go
@@ -36,7 +36,6 @@ func newDestroyCmd() *cobra.Command {
 			if preview || yes ||
 				confirmPrompt("This will permanently destroy all resources in the '%v' environment!", envName.String()) {
 				return lumiEngine.Destroy(envName, engine.DestroyOptions{
-					Package:  pkgargFromArgs(args),
 					DryRun:   preview,
 					Debug:    debug,
 					Parallel: parallel,

--- a/cmd/preview.go
+++ b/cmd/preview.go
@@ -19,7 +19,7 @@ func newPreviewCmd() *cobra.Command {
 	var showSames bool
 	var summary bool
 	var cmd = &cobra.Command{
-		Use:   "preview [<package>] [-- [<args>]]",
+		Use:   "preview",
 		Short: "Show a preview of updates to an environment's resources",
 		Long: "Show a preview of updates an environment's resources\n" +
 			"\n" +
@@ -30,8 +30,8 @@ func newPreviewCmd() *cobra.Command {
 			"determine what operations must take place to achieve the desired state. No changes to the\n" +
 			"environment will actually take place.\n" +
 			"\n" +
-			"By default, the package to execute is loaded from the current directory. Optionally, an\n" +
-			"explicit path can be provided using the [package] argument.",
+			"The package to execute is loaded from the current directory. Use the `-C` or `--cwd` flag to\n" +
+			"use a different directory.",
 		Run: cmdutil.RunFunc(func(cmd *cobra.Command, args []string) error {
 			envName, err := explicitOrCurrent(env)
 			if err != nil {
@@ -39,7 +39,6 @@ func newPreviewCmd() *cobra.Command {
 			}
 
 			return lumiEngine.Preview(envName, engine.PreviewOptions{
-				Package:              pkgargFromArgs(args),
 				Debug:                debug,
 				Analyzers:            analyzers,
 				Parallel:             parallel,

--- a/cmd/pulumi.go
+++ b/cmd/pulumi.go
@@ -31,9 +31,17 @@ func NewPulumiCmd() *cobra.Command {
 	var logFlow bool
 	var logToStderr bool
 	var verbose int
+	var cwd string
 	cmd := &cobra.Command{
 		Use: "pulumi",
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if cwd != "" {
+				err := os.Chdir(cwd)
+				if err != nil {
+					cmdutil.ExitError(err.Error())
+				}
+			}
+
 			cmdutil.InitLogging(logToStderr, verbose, logFlow)
 		},
 		PersistentPostRun: func(cmd *cobra.Command, args []string) {
@@ -41,6 +49,7 @@ func NewPulumiCmd() *cobra.Command {
 		},
 	}
 
+	cmd.PersistentFlags().StringVarP(&cwd, "cwd", "C", "", "Run pulumi as if it had been started in another directory")
 	cmd.PersistentFlags().BoolVar(&logFlow, "logflow", false, "Flow log settings to child processes (like plugins)")
 	cmd.PersistentFlags().BoolVar(&logToStderr, "logtostderr", false, "Log to stderr instead of to files")
 	cmd.PersistentFlags().IntVarP(
@@ -54,14 +63,6 @@ func NewPulumiCmd() *cobra.Command {
 	cmd.AddCommand(newVersionCmd())
 
 	return cmd
-}
-
-func pkgargFromArgs(args []string) string {
-	if len(args) > 0 {
-		return args[0]
-	}
-
-	return ""
 }
 
 func confirmPrompt(msg string, name string) bool {

--- a/cmd/update.go
+++ b/cmd/update.go
@@ -20,7 +20,7 @@ func newUpdateCmd() *cobra.Command {
 	var showSames bool
 	var summary bool
 	var cmd = &cobra.Command{
-		Use:   "update [<package>] [-- [<args>]]",
+		Use:   "update",
 		Short: "Update the resources in an environment",
 		Long: "Update the resources in an environment\n" +
 			"\n" +
@@ -31,8 +31,8 @@ func newUpdateCmd() *cobra.Command {
 			"must take place to achieve the desired state. This command results in a full snapshot of the\n" +
 			"environment's new resource state, so that it may be updated incrementally again later.\n" +
 			"\n" +
-			"By default, the package to execute is loaded from the current directory. Optionally, an\n" +
-			"explicit path can be provided using the [package] argument.",
+			"The package to execute is loaded from the current directory. Use the `-C` or `--cwd` flag to\n" +
+			"use a different directory.",
 		Run: cmdutil.RunFunc(func(cmd *cobra.Command, args []string) error {
 			envName, err := explicitOrCurrent(env)
 			if err != nil {
@@ -40,7 +40,6 @@ func newUpdateCmd() *cobra.Command {
 			}
 
 			return lumiEngine.Deploy(envName, engine.DeployOptions{
-				Package:              pkgargFromArgs(args),
 				Debug:                debug,
 				DryRun:               dryRun,
 				Analyzers:            analyzers,

